index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Persaonal Portfolio</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css"
+      integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@600&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <nav>
+      <ul>
+        <li><h1 class="welcome">Welcome</h1></li>
+
+        <li><i class="fa-solid fa-bars fa-2x"></i></li>
+      </ul>
+    </nav>
+    <header class="header-section">
+      <h2 class="header-section-title">
+        <p class="header-name">Hey there. I’m Coral</p>
+        <p class="header-name header-title">I’m a software developer</p>
+      </h2>
+    </header>
+    <p class="paragraph">
+      I can help you build a product , feature or website Look through some of
```

Comment:
- [ ] For semantic purposes, I suggest following the structure below while also using flexbox. :point_down: 
```

<header><a>Logo</a><nav><img src="humburger"></nav><
/header>
<section>
<h1>Title</h1>
<h2>Title2</h2>
<p>text ......</p>
<ul>
for the social icons
</ul>
</section>
```


- [ ] :thinking: Kindly make sure to add the background image as shown in  the Figma below, to match the design:

- Your design:
![Screenshot from 2023-01-19 13-14-40](https://user-images.githubusercontent.com/97834160/213433488-93a5b883-db01-4234-887f-5e0347f0c1e3.png)

- Figma design:
![Screenshot from 2023-01-19 13-43-19](https://user-images.githubusercontent.com/97834160/213433897-b1662017-d6c8-468a-ba24-564c967425b1.png)

Reply:
- I think some things may have gone wrong with your refactoring. On my browser, your mobile stup doesn't seem to be properly designed. Can we fix that? 

Here is what I could see compared to Figma 👇🏽 
| Your Design  |Figma Design |
| :---:   | :---: | 
| <img width="393" alt="Screenshot 2023-01-19 at 16 05 23" src="https://user-images.githubusercontent.com/99246359/213573781-2bd935b4-ad66-4dc8-bc99-17b7c6e0a831.png"> |<img width="346" alt="Screenshot 2023-01-19 at 16 13 47" src="https://user-images.githubusercontent.com/99246359/213574100-b5242fdb-09d6-4ac5-b3d9-16626e9836be.png">  |